Test/Mono.Cecil.Tests.csproj

```diff
@@ -7,6 +7,10 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp">
       <Version>2.10.0</Version>
     </PackageReference>
+    <PackageReference Include="runtime.$(NetCoreSdkPortableRuntimeIdentifier).Microsoft.NETCore.ILAsm">
+      <Version>9.0.4</Version>
```

Comment:
Not sure what the repo convention for cecil is but we usually avoid hardcoding versions.

Reply:
this is just a test and any ilasm version from the last decade would work so this is fine.